scania-digital-design-system/sdds
# AB#  Added spinner web component

**Describe pull-request**  
_Describe what the pull-request is about_
A spinner web component has added.
Added a web component to the demo/angular and demo/react folders.
**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #

**How to test**  
_Add description how to test if possible_
1. Go to... component/spinner
2. Check in...
3. Run ...

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
